src/fides/api/ctl/routes/generate.py

```diff
@@ -142,7 +142,9 @@ async def generate(
                 aws_config=generate_request_payload.generate.config,
                 organization=organization,
             )
-        elif generate_request_payload.generate.target.lower() == "db":
+        elif generate_request_payload.generate.target.lower() == "db" and isinstance(
+            generate_request_payload, DatabaseConfig
```

Comment:
This was my bad in the previous suggestion.
```suggestion
            generate_request_payload.generate.config, DatabaseConfig
```